provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/upgrade-bridge.yml

```
@@ -12,6 +12,17 @@ on:
         required: false
         type: string
         default: "latest"
+      target-pulumi-version:
+        description: |
+             Set the version of `pulumi/pkg` and `pulumi/sdk` to depend on for bridged providers. Currently,
+             these versions inform the linked runtime and SDK generation in all languages except Java. Valid
+             options are:
+             - "", "follow": Use the same version as pulumi-terraform-bridge
```

The final implementation doesn't support `"follow"`.
```suggestion
             - "": Use the same version as pulumi-terraform-bridge
```